docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md

```diff
@@ -0,0 +1,99 @@
+## CTCGreedyDecoderSeqLen <a name="CTCGreedyDecoderSeqLen"></a> {#openvino_docs_ops_sequence_CTCGreedyDecoderSeqLen_6}
+
+**Versioned name**: *CTCGreedyDecoderSeqLen-6*
+
+**Category**: Sequence processing
+
+**Short description**: *CTCGreedyDecoderSeqLen* performs greedy decoding of the logits provided as the first input. The sequence lengths are provided as the second input.
+
+**Detailed description**:
+
+This operation is similar to the [TensorFlow CTCGreedyDecoder](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_greedy_decoder).
+
+The operation *CTCGreedyDecoderSeqLen* implements best path decoding.
+Decoding is done in two steps:
+
+1. Concatenate the most probable classes per time-step which yields the best path.
+
+2. Remove duplicate consecutive elements if the attribute *merge_repeated* is true and then remove all blank elements.
+
+Sequences in the batch can have different length. The lengths of sequences are coded in the second input integer tensor `sequence_length`.
+
+The main difference between [CTCGreedyDecoder](CTCGreedyDecoder_1.md) and CTCGreedyDecoderSeqLen is in the second input. CTCGreedyDecoder uses 2D input floating point tensor with sequence masks for each sequence in the batch while CTCGreedyDecoderSeqLen uses 1D integer tensor with sequence lengths.
+
+**Attributes**
+
+* *merge_repeated*
+
+  * **Description**: *merge_repeated* is a flag for merging repeated labels during the CTC calculation. If the value is false the sequence `ABB*B*B`  (where '*' is the blank class) will look like `ABBBB`. But if the value is true, the sequence will be `ABBB`.
+  * **Range of values**: true or false
+  * **Type**: `boolean`
+  * **Default value**: true
+  * **Required**: *no*
+    
+* *classes_index_type*
+
+  * **Description**: the type of output tensor with classes indices
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i32"
+  * **Required**: *No*
+  
+* *sequence_length_type*
+
+  * **Description**: the type of output tensor with sequence length
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i32"
+  * **Required**: *No*
```

Comment:
I think we no longer use optional attributes in the specification, don't we, @lazarevevgeny ?

Reply:
We still may have optional attributes in the spec. But MO should always generate values in the IR even for optional attributes.